portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Task {
+
+  private final Long id;
+  private final String name;
+  private final String date;
+  private final String text;
+
+  public Task(Long num, String username, String textDate, String words) {
+    id = num;
+    name = username;
+    date = textDate;
+    text = words;
+  }
```

Comment:
These variable names are confusing.

Lets just use id, username, date, msg (description/desc)?

In java the parameters name can be the same as the variable names.
So you'd assign them something like

this.id = id;
this.username = username;

Please update it below as well, where you call this Task.